### DISCUSSION TITLE
[274190] 武装原型【Broforce】

### DISCUSSION THREAD

**Discussion by miaor123** [pjy612/SteamManifestCache#476] [Share]
`appid:274190`
keys:
```
274191;995563ca038cedb11588f1581c6227e191f0d933b321007dfb5af38b067a22e1
```

[274190.zip](https://github.com/pjy612/SteamManifestCache/files/14209198/274190.zip)

SAI找不到了因此没有CODE。。
邮箱：[2814589496@qq.com](mailto:2814589496@qq.com)

**Comment by pjy612** [comment]
* 麻烦 按照格式 #26

**Reply by miaor123** [reply]
已修改
